Makefile

```diff
@@ -101,21 +101,18 @@ test-update: update-prepare update
 		exit 1; \
 	} \
 
-test-e2e: image hyperkube
-	docker push $(IMAGE)
-	docker build -t $(TEST_IMAGE) \
-		--build-arg K8S_VERSION=$(K8S_VERSION) \
-		--build-arg AKSENGINE_VERSION=$(AKSENGINE_VERSION) \
-		tests/k8s-azure
-	docker run --env-file $(K8S_AZURE_ACCOUNT_CONFIG) \
-		-e K8S_AZURE_TEST_ARTIFACTS_DIR=$(ARTIFACTS) \
-		-v $(WORKSPACE):$(WORKSPACE) \
-		-v $(ARTIFACTS):$(ARTIFACTS) \
-		$(TEST_IMAGE) e2e -v -caccm_image=$(IMAGE) \
-		-ctype=$(SUITE) \
-		-csubject=$(SUBJECT) \
-		-cmanifest=$(TEST_MANIFEST) \
-		-chyperkube_image=$(HYPERKUBE_IMAGE)
+test-e2e: deploy
+	cd $GOPATH/src/k8s.io/kubernetes
```

Comment:
After moving this into a script, please add a check of $GOPATH/src/k8s.io/kubernetes. If it doesn't exist, we should git clone it first.